linker version-script in tutorial does not work with OpenSSL 1.1

Today Maciej Fijałkowski pointed out that [Using your own OpenSSL on Linux](https://cryptography.io/en/latest/installation.html#using-your-own-openssl-on-linux) tutorial no longer works. Building fails with ``duplicate expression `*' in version information``.

I assume it's due to the fact that OpenSSL 1.1 now uses its own version script ``libcrypto.map``.

```
$ cat > openssl.ld << EOF
OPENSSL_1.1.1I_CUSTOM {
    global:
        *;
};
EOF
$ ./config -Wl,--version-script=openssl.ld -Wl,-Bsymbolic-functions -fPIC shared
Operating system: x86_64-whatever-linux2
Configuring OpenSSL version 1.1.1i (0x1010109fL) for linux-x86_64
Using os-specific seed configuration
Creating configdata.pm
Creating Makefile
```

```
$ make
...
gcc -fPIC -pthread -m64 -Wa,--noexecstack -Wall -O3 -fPIC -L. -Wl,-znodelete -shared -Wl,-Bsymbolic  -Wl,--version-script=openssl.ld -Wl,-Bsymbolic-functions -Wl,-soname=libcrypto.so.1.1 \
        -o libcrypto.so.1.1 -Wl,--version-script=libcrypto.map ...
...
/usr/bin/ld: duplicate expression `*' in version information
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:756: libcrypto.so] Error 1

--- a/docs/installation.rst
+++ b/docs/installation.rst
@@ -143,17 +143,7 @@ this when configuring OpenSSL:
 
 .. code-block:: console
 
-    $ ./config -Wl,--version-script=openssl.ld -Wl,-Bsymbolic-functions -fPIC shared
-
-You'll also need to generate your own ``openssl.ld`` file. For example::
-
-    OPENSSL_1.1.0E_CUSTOM {
-        global:
-            *;
-    };
-
-You should replace the version string on the first line as appropriate for your
-build.
+    $ ./config -Wl,-Bsymbolic-functions -fPIC shared
 
 Static Wheels
 ~~~~~~~~~~~~~

